bluesky_widgets/models/run_engine_client.py

```diff
@@ -50,15 +50,43 @@ class RunEngineClient:
     def __init__(self, worker_address=None):
         self._client = ZMQCommSendThreads(zmq_server_address=worker_address)
 
+        # User name and group are hard coded for now
+        self._user_name = "GUI Client"
+        self._user_group = "admin"
+
         self._re_manager_status = {}
         self._re_manager_connected = None
         self._re_manager_status_time = time.time()
         # Minimum period of status update (avoid excessive call frequency)
         self._re_manager_status_update_period = 0.2
 
+        self._allowed_devices = {}
+        self._allowed_plans = {}
+        self._plan_queue_items = []
```

Comment:
Can the _plan queue_ and _plan history_ be separated out as a stand-alone models? The `RunEngineClient` has grown quite large. It might be easier to read and to test if these were separate, as suggested by

https://github.com/bluesky/bluesky-widgets/pull/123/files#diff-f3dfb705388c1ed4439be76ac52bad8fb23761b11a9ed57ef18acfee532837f9R41-R46

But if there is some reason why this would be difficult we can leave it for now....

Reply:
I would prefer to keep it as a single model for now, because it is not clear to me at this point how queue and history models should interact. But I am trying to keep the code separate as much as possible, so that it would be more readable and could be easily separated if necessary.

Reply:
OK, I can live with that. :+1: